crates/dojo-core/src/model.cairo

```diff
@@ -1,26 +1,51 @@
+use dojo::world::IWorldDispatcher;
 use starknet::SyscallResult;
 
+#[derive(Serde, Drop, Copy, PartialEq)]
+enum ModelLayoutType {
+    Serial,
+    Hash    
```

Comment:
For the record as we've discussed, let's try to have the inner type here to remove one enum level into the serialized form. 👍

ah, seems that `ModelLayout` is now the new one actually.